### DISCUSSION TITLE
Trusted Types production level support

### DISCUSSION THREAD

**Discussion by lzomlot** [jquery/jquery#5526] [Q&A] [answered]
### Discussed in https://github.com/jquery/jquery/discussions/5520

<div type='discussions-op-text'>

<sup>Originally posted by **lzomlot** July 17, 2024</sup>
Our service's Trusted-Types adoption is blocked by jQuery. We are aware that you added support for Trusted-Types support v4.0 beta. Thank you for that :). What is the ETA for v4 production release that supports trusted types?</div>

**Comment by mgol** [comment, marked as answer]
<sub>I migrated this to a discussion.</sub>

jQuery 4 is generally feature-complete, the blockers for the final release boil down to preparing an upgrade guide and patching all relevant APIs in jQuery Migrate. We might also do an RC before the final version.

I cannot promise any timelines, though.
